plugins/azure-devops/src/components/PullRequestsPage/lib/utils.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DashboardPullRequest,
+  PullRequestVoteStatus,
+  Reviewer,
+} from '@backstage/plugin-azure-devops-common';
+import {
+  PullRequestFilter,
+  PullRequestGroup,
+  PullRequestGroupConfig,
+} from './types';
+
+/**
+ * Creates a filter that matches pull requests created by `userEmail`.
+ * @param userEmail an email to filter by.
+ * @returns a filter for pull requests created by `userEmail`.
+ */
+export function getCreatedByUserFilter(
+  userEmail: string | undefined,
+): PullRequestFilter {
+  return (pullRequest: DashboardPullRequest): boolean =>
+    pullRequest.createdBy?.uniqueName?.toLocaleLowerCase() ===
```

Comment:
Intention is to have these use a hardcoded locale right? So `toLocaleLowerCase('en-US')`

Reply:
I would use `toLowerCase()` but there is a linting error. Should I just be using `toLocaleLowerCase('en-US')` in all cases?

I won't fix it in this PR as I have plans to delete that code anyway but I will take onboard the outcome of this discussion for future changes 🙂

Do you know the reasoning behind this linting rule?:
> Avoid using .toLowerCase(), use .toLocaleLowerCase('en-US') instead. This rule can sometimes be ignored when converting text to be displayed to the user.eslintno-restricted-syntax

Does backstage have plans to support localization?

Reply:
So as the comment says, if it's for displaying regular text to users, then you are right in wanting to use `toLowerCase` or `toLocaleLowerCase`. The reason that this lint rule is in place, is because there are several situations - in particular for technical identifiers - that you absolutely must avoid the user-locale-dependent casing functions. If you do `toLowerCase` in a Turkish locale for example, you may find that their [experiences break](http://haacked.com/archive/2012/07/05/turkish-i-problem-and-why-you-should-care.aspx/) because their lowercase of the letter i turns out to be a different character [without a dot](https://en.wikipedia.org/wiki/Dotless_I), so their `metadata.name` of an entity results in a 404 all of the sudden.

We do want to support i18n for sure, but have quite some ways to go before doing that well. This lint rule in particular, however, is not meant to counteract that. It can be annoying, but given the current code base, it will more often be right than wrong, and ignore comments can be used to silence it when you are sure that it was wrong.